src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -273,6 +273,11 @@ internal static class Defaults
 
         PredictionKind ITrainer.PredictionKind => PredictionKind.Recommendation;
 
+        /// <summary>
+        /// The <see cref="TrainerInfo"/> contains general parameters for this trainer.
```

Comment:
Seems that you just do a useless copy-paste here (see the change below). Could you just recover it back since it's unnecessary